test/server/cards/01_SOR/units/ZebOrreliosHeadstrongWarrior.spec.ts

```diff
@@ -0,0 +1,53 @@
+describe('Zeb Orrelios, Headstrong Warrior', function () {
+    integration(function (contextRef) {
+        describe('Zeb Orrelios\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['zeb-orrelios#headstrong-warrior', 'swoop-racer'],
+                        spaceArena: ['green-squadron-awing']
+                    },
+                    player2: {
+                        groundArena: ['superlaser-technician', 'consular-security-force', 'steadfast-battalion'],
+                    }
+                });
+            });
+
+            it('should deal 4 damage to a ground unit when he kill someone and survives', function () {
+                const { context } = contextRef;
+
+                // kill superlaser technician, should deal 4 to a ground unit
+                context.player1.clickCard(context.zebOrrelios);
+                context.player1.clickCard(context.superlaserTechnician);
```

Comment:
Can we switch to a unit without a defeat prompt to make this simpler

Reply:
No, I want to test if a unit doesn't go to discard the ability still works